lti_consumer/tests/unit/plugin/test_views_lti_deep_linking.py

```diff
@@ -152,3 +152,52 @@ def test_lti_deep_linking_valid_request(self):
             },
         )
         self.assertEqual(response.status_code, 200)
+
+
+class LtiDeepLinkingContentEndpointTestCase(LtiDeepLinkingTestCase):
+    """
+    Test ``deep_linking_content_endpoint`` view.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.url = '/lti_consumer/v1/lti/{}/lti-dl/content'.format(self.lti_config.id)
+
+    @patch('lti_consumer.plugin.views.has_access', return_value=False)
+    def test_forbidden_access(self, has_access_patcher):  # pylint: disable=unused-argument
+        """
+        Test if throws 403 when user doesn't have access.
```

Comment:
Test if 403 is returned when a user doesn't have access.

Reply:
Updated. ``throws`` kind of means an exception, but in this case ``returned`` is a more appropriate word.